server/services/withings/lib/utils/withings.buildBatteryLevelValues.js

```diff
@@ -17,11 +17,11 @@ function buildBatteryLevelValues(currentBatValueString) {
       break;
     case 'medium':
       batValueString = `${currentBatValueString} (> 30%)`;
-      batValue = 30;
-      break;
+      batValue = 50;
+      break; 
     case 'high':
       batValueString = `${currentBatValueString} (> 75%)`;
```

Comment:
You changed the batValue from 75 to 100, but not the text!

Reply:
i fix this